packages/renderer/src/AppNavigation.spec.ts

```diff
@@ -1,5 +1,5 @@
 /**********************************************************************
- * Copyright (C) 2023 Red Hat, Inc.
+ * Copyright (C) 2023,2024 Red Hat, Inc.
```

Comment:
```suggestion
 * Copyright (C) 2023-2024 Red Hat, Inc.
```
?

Reply:
Sorry, probably a misnomer from when I was originally told how to do it. Fixed.